urdf_parser/test/memtest.cpp

```diff
@@ -3,6 +3,7 @@
 #include <iostream>
 
 int main(int argc, char** argv){
+  (void) argc;
```

Comment:
Actually, rather than doing this, I think we should fix this `main` up, since it is buggy.

That is, if it ever got called with no arguments, it would index past the end of argv below and crash.

My suggestion here is to add:

```
if (argc != 2) {
  fprintf(stderr, "Usage: %s <urdf>\n", argv[0]);
  return 1;
}
```

And also add a `return 0;` to the end of the function for completeness.